### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by GRAWek19** [NicholasDawson/Data-Extractor-for-AniList#1]
When I try to run there appears to be something wrong with line 55

**Comment by NicholasDawson** [comment]
Yeah after looking into it you are correct, I think they changed something with how their page loads. If you want to try to fix I'll gladly accept a PR, but I can't put any time into fixing this unfortunately :(

**Comment by GRAWek19** [comment]
What's interesting it worked ONCE and stopped working again.
I spent a bit trying to figure it out but my knowledge as a programmer is really basic so not that much i can do :/
